custom_components/opensprinkler/__init__.py

```diff
@@ -19,7 +18,7 @@
 from homeassistant.util import slugify, Throttle
 from homeassistant.util.dt import utc_from_timestamp
 
-from .const import DEFAULT_PORT, DOMAIN, SCAN_INTERVAL
+from .const import DEFAULT_PORT, DOMAIN, UPDATE_INTERVAL
```

Comment:
I would personally leave the original var name SCAN_INTERVAL, it is a standard across hass components

Reply:
I am not sure how python works but I wanted to keep it in one place so can import it, it is set as `SCAN_INTERVAL` below

Reply:
changed it slightly

Reply:
You could import it as SCAN_INTERVAL, that worked in my fix.. if you want to keep name UPDATE_INTERVAL see my recommendation below

Reply:
nah it that works I will change it all to SCAN_INTERVAL. Not very experienced with python so bear with me :)